src/main.zig

```diff
@@ -401,6 +401,14 @@ const usage_build_generic =
     \\  -fno-allow-shlib-undefined     Disallows undefined symbols in shared libraries
     \\  --eh-frame-hdr                 Enable C++ exception handling by passing --eh-frame-hdr to linker
     \\  --emit-relocs                  Enable output of relocation sections for post build tools
+    \\  -z [arg]                       Append linker arguments
```

Comment:
Hmm, `-z <arg>` seems specific to ELF linkers; would it make more sense to expose each flag of interest separately with `-fnotext` or something similar? Also, we should definitely point out these are ELF-specific. @andrewrk I'd appreciate your look at this too!

Reply:
`-z` specifies an extension flag. `wasm-ld` supports them as well and we actually already make use of it to set the stack size for executables. So if we go with this, perhaps those should be supported too.

Reply:
```suggestion
    \\  -z [arg]                       Set linker extension flags
```

Reply:
```suggestion
    \\  -z [arg]                       Set linker extension flags
```